physics/Magnetized Cold Plasma Properties.ipynb

```diff
@@ -0,0 +1,272 @@
+      "text/plain": [
+       "<Figure size 864x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(12, 6))\n",
+    "plt.semilogx(f, abs(S_pos),\n",
+    "             f, abs(D_pos),\n",
+    "             f, abs(P_pos), lw=2)\n",
+    "plt.semilogx(f, abs(S_neg), '#1f77b4',\n",
+    "             f, abs(D_neg), '#ff7f0e',\n",
+    "             f, abs(P_neg), '#2ca02c', lw=2, ls='--')\n",
+    "plt.yscale('log')\n",
+    "plt.grid(True, which='major')\n",
+    "plt.grid(True, which='minor')\n",
+    "plt.ylim(1e-4, 1e8)\n",
+    "plt.xlim(1e6, 200e9)\n",
+    "plt.legend(('S > 0', 'D > 0', 'P > 0', 'S < 0', 'D < 0', 'P < 0'),\n",
+    "           fontsize=16, ncol=2)\n",
+    "plt.xlabel('RF Frequency [Hz]', size=16)\n",
+    "plt.ylabel('Absolute value', size=16)\n",
+    "plt.tick_params(labelsize=14)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Cold Plasma tensor elements in the rotating basis"
```

Comment:
this spot could also use a really brief description. Am I right saying that those would be left-handed and right-handed circularly polarized waves?

Reply:
Yes indeed.